packages/next/src/client/components/errors/use-error-handler.ts

```diff
@@ -70,8 +70,8 @@ export function useErrorHandler(
       )
 
       // Reset error queues.
-      errorQueue.splice(0, 0)
-      rejectionQueue.splice(0, 0)
+      errorQueue.splice(0, errorQueue.length)
```

Comment:
This is not critical for the test, if you remove the change here tests can also pass. Just change it to make it correctly reset the error queue